src/langsmith/hosting.mdx

```diff
@@ -7,7 +7,7 @@ mode: wide
 This section covers how to host and manage **LangSmith**: the infrastructure, services, and datastores that power your observability, evaluation, and agent deployment capabilities.
 
 <Callout icon="building" color="#2563EB" iconType="regular">
-**Start here if you're setting up or maintaining LangSmith infrastructure.** Looking to deploy your applications? See the [Deployment section](/langsmith/deployments).
+**Start here if you're setting up or maintaining LangSmith infrastructure.** If you're ready to deploy your application, the [Deployment section](/langsmith/deployments) covers application structure, deployment configuration, and guides for deploying to Cloud.
```

Comment:
Suggestion: `If you're ready to deploy your application` --> `If you're ready to deploy your agent application`

Just a to disambiguate what `application` refers to in the context of the `LangSmith Hosting` tab.